menas/ui/service/SchemaFieldSelector.js

```diff
@@ -106,12 +111,23 @@ class SchemaFieldSelector {
     return rawTitle.match(/<strong>(?<title>.+)<\/strong>/).groups.title;
   }
 
+  reset(tree) {
+    // This is a workaround for a bug in the Tree component of 1.56.x and 1.58.x
+    // which throws "Cannot read property 'setSelectedIndex' of undefined" error
+    // TODO: verify whether this was fixed in the subsequent versions
```

Comment:
Create an Issue as we talked about?

Reply:
The workaround is good enough as is. Since we can't really fix it we don't need a task for it. And I personally don't intend to keep track of when this is fixed if ever so I don't need the TODO either.